src/gui/sdlmain.cpp

```diff
@@ -4223,6 +4219,12 @@ static void config_add_sdl() {
 	        "  vfr:   Always present changed DOS frames at a variable frame rate.");
 	pstring->Set_values(presentation_modes);
 
+	pint = sdl_sec->Add_int("vfr_minimum_rate", always, 7);
+	pint->Set_help(
+	        "Set VFR presentation mode's minimum refresh rate, in Hz (7 by default).\n"
+	        "Note the DOS rate determines the maximum VFR rate; this does not override it.");
+	pint->SetMinMax(1, RefreshRateMax);
```

Comment:
Ah, ok, so we've already provided a valid value range here. Then we should perhaps document it in the description?

Reply:
(ditto as above; conf setting being knocked out)